rpcd/playbooks/rpc-pre-upgrades.yml

```diff
@@ -19,18 +19,63 @@
   roles:
     - { role: "rpc_pre_upgrade", tags: [ "rpc-pre-upgrades" ] }
 
-- name: restart beaver
+- name: remove beaver
   hosts: all
   tags:
     - rpc-pre-upgrades
+    - remove-beaver
   tasks:
     - name: Check if beaver is installed
       stat:
         path: /etc/init.d/beaver
       register: beaver_init_script
+      changed_when: false
```

Comment:
@Apsu Good call, although you may wish to reference https://github.com/ansible/ansible-modules-core/blob/stable-1.9/files/stat.py#L351 instead, seeing as that's the Ansible version being used. :)